opm/input/eclipse/EclipseState/Grid/FieldData.hpp

```diff
@@ -22,92 +22,110 @@
 
 #include <opm/input/eclipse/EclipseState/Grid/Box.hpp>
 #include <opm/input/eclipse/EclipseState/Grid/Keywords.hpp>
+
 #include <opm/input/eclipse/Deck/value_status.hpp>
 
-#include <string>
-#include <vector>
-#include <optional>
-#include <array>
 #include <algorithm>
+#include <array>
+#include <cstddef>
+#include <optional>
 #include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace Opm { namespace Fieldprops {
```

Comment:
since there only are Opm::FieldProps members this can be namespace Opm::Fieldprops